### PR TITLE
Fix bisection regenerated issue no-baseline error when baseline exists

### PR DESCRIPTION
**Pull request by ewlu**
[get_most_recent_ci_hash.py](https://github.com/patrick-rivos/riscv-gnu-toolchain/blob/build-frequent/scripts/get_most_recent_ci_hash.py) was encountering an error where it could not retrieve the 100 most recent commits due to an error with the tree. Pull the most updated version of gcc in the regenerate summary to fix

Test: https://github.com/ewlu/riscv-gnu-toolchain/actions/runs/6006001115/job/16307721347?pr=41
Regenerate: https://github.com/patrick-rivos/riscv-gnu-toolchain/actions/runs/6017124614

### CODE REVIEW

File: .github/workflows/generate-summary.yaml [hunk 1]
```diff
@@ -261,6 +261,37 @@ jobs:
...
+            pk
+            qemu
+          key: submodules
+
+      - name: Initalize gcc
+        if: steps.retrieve-cache.outputs.cache-hit != 'true'
+        run: |
+          rm -rf gcc
+          git clone git://gcc.gnu.org/git/gcc.git
+          git submodule update --recursive --progress --recommend-shallow
```
Comment (patrick-rivos) [inline .github/workflows/generate-summary.yaml:286]: Since we only use GCC, I think we can remove this line.

**Review by patrick-rivos — CHANGES_REQUESTED** [review]
One minor change, LGTM otherwise

File: .github/workflows/generate-summary.yaml (hunk 1 above)
Comment (ewlu) [reply .github/workflows/generate-summary.yaml:286]: testing regenerate: https://github.com/patrick-rivos/riscv-gnu-toolchain/actions/runs/6090823639
other test: https://github.com/ewlu/riscv-gnu-toolchain/actions/runs/6090811474

**Review by patrick-rivos — APPROVED** [review]
LGTM - I'm just gonna remove the commented out line then merge it.

**Pull request merged**